apps/src/tutorialExplorer/roboticsButton.jsx

```diff
@@ -61,6 +61,7 @@ export default class RoboticsButton extends React.Component {
                 <img
                   src="/images/learn/robotics-link.png"
                   style={styles.roboticsButtonImage}
+                  alt="A humanoid robot"
```

Comment:
This should come from the `i18n` string table.

Reply:
Looking at this again in context, I think it makes more sense to set it to alt="". The image doesn't really have any semantic value.